sdcm/logcollector.py

```diff
@@ -102,9 +102,10 @@ class BaseLogEntity:  # pylint: disable=too-few-public-methods
     collect_timeout = 300
     _params = {}
 
-    def __init__(self, name, command="", search_locally=False):
+    def __init__(self, name, command="", search_locally=False, node_attr=None):
         self.name = name
         self.cmd = command
+        self.node_attr = node_attr
```

Comment:
I would suggest doing it a bit more general so multiple attributes can be formatted into the command
Maybe we can format the node itself ?
```python
>>> cmd = "cqlsh {node.ip}"
>>>
>>> class Node: pass
...
>>> n = Node()
>>> n.ip = "1234"
>>>
>>> cmd.format(node=n)
'cqlsh 1234'
```

Reply:
Changed it - prepare the files during tearDown